src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.scss

```diff
@@ -1,3 +1,12 @@
 .icon-warning-color {
   margin-left: 3px;
 }
+
+.item-action-btn {
+  margin-top: 1.5rem;
+}
```

Comment:
i don't think we need it. We are already defining it here: https://github.com/ceph/ceph/blob/main/src/pybind/mgr/dashboard/frontend/src/styles/ceph-custom/_forms.scss#L166